<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-3O5LJFC" failures="5" tests="6" name="BankingApp Test(failed)" time="25.903" errors="0" timestamp="2021-09-15T17:17:26 PDT">
  <testcase classname="com.inetbanking.testCases.TC_LoginTest_001" name="loginTest" time="0.62"/>
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="3.807">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="3.89">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="3.743">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="3.729">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="3.749">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- BankingApp Test(failed) -->
